sdk/content-components/documentation/docs/api/getSASJobExecutionUrl.md

```diff
@@ -0,0 +1,28 @@
+---
+id: getSASJobExecutionUrl
+title: getSASJobExecutionUrl
+---
+
+```
+getSASJobExecutionUrl(item, url): Promise<string | undefined>
+```
+
+Given a target job definition item, such as from an `onSelect` event, and a target deployment URL, generate a SAS Job Execution URL that will open the job and allow it to be executed if it is a job with an HTML prompt. This URL can be opened in a new tab or window, or embedded in an iframe if allowed by the security settings on the deployment.
```

Comment:
This function generates a SAS Job Execution URL. You can use the URL to open a job and execute it if it is a job with an HTML prompt. This URL can be opened in a new tab or window, or embedded in an iframe if allowed by the security settings on the deployment.